mathosphere-core/src/main/java/com/formulasearchengine/mathosphere/mlp/FlinkMlpRelationFinder.java

```diff
@@ -213,7 +211,7 @@ public void reduce(Iterable<ParsedWikiDocument> iterable, Collector<String> coll
         System.err.println("fn:" + fnOverall.size());
         System.err.println("tp:" + tpOverall.size());
 
-        System.err.println("Overall definition evaluation");
+        System.err.println("Overall definition evaluation - by this method, better use evaluation in Evaluation package.");
```

Comment:
since you are refactoring the code anyhow... maybe you can try to replace the sysouts with LOG statments